client/src/components/Ratings/index.jsx

```diff
@@ -78,51 +63,46 @@ function addStarAndBar(results, stars) {
         top: 4,
       }}
       />
-
       <div style={progressStyleObj} />
-
       <br />
     </div>
   );
 }
 
-function Ratings({ results, resultsMeta, setReviews, sort }) {
+function Ratings({
+  results, resultsMeta, setReviews, sort
+}) {
 
   return (
     <div id="ratings-reviews">
-
       <div>Ratings And Reviews</div>
-
       <div className="review-summary">
-
         <h2 id="big-review-num">
           <div style={{ float: 'left' }}>{averageToNearestTenth(results) ? averageToNearestTenth(results) : ""}</div>
           <div id="review-stars"><Stars rating={averageToNearestTenth(results)} /></div>
         </h2>
-
         <div>
           {getPercentageRecommended(results)
             ? `${getPercentageRecommended(results)}% of reviews recommend this product`
             : "Loading..."}
         </div>
         <br />
         <div>
-          <div onClick={() => filterByRating(5)} style={{ padding: 10 }}>{addStarAndBar(results, 5)}</div>
+          <div className="star-bar">{addStarAndBar(results, 5)}</div>
           <br />
-          <div onClick={() => filterByRating(4)} style={{ padding: 10 }}>{addStarAndBar(results, 4)}</div>
+          <div className="star-bar">{addStarAndBar(results, 4)}</div>
           <br />
-          <div onClick={() => filterByRating(3)} style={{ padding: 10 }}>{addStarAndBar(results, 3)}</div>
+          <div className="star-bar">{addStarAndBar(results, 3)}</div>
           <br />
-          <div onClick={() => filterByRating(2)} style={{ padding: 10 }}>{addStarAndBar(results, 2)}</div>
+          <div className="star-bar">{addStarAndBar(results, 2)}</div>
           <br />
-          <div onClick={() => filterByRating(1)} style={{ padding: 10 }}>{addStarAndBar(results, 1)}</div>
+          <div className="star-bar">{addStarAndBar(results, 1)}</div>
         </div>
```

Comment:
applying one class to the parent `<div>` then using the selector `.star-bar div` is a better practice here
also specify a margin instead of using `<br />`
also also don't use `onClick` with non responsive tags. Use a button or an anchor tag. This semantic distinction is important for screen readers